sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/CodeSnippetsAysnc.java

```diff
@@ -0,0 +1,146 @@
+package com.azure.data.tables;
+
+
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+
+
+public class CodeSnippetsAysnc {
+
+
+    public void AsyncSnippets() {
+
+        //build service client
+        TableServiceAsyncClient tableServiceAsyncClient = new TableServiceAsyncClientBuilder()
```

Comment:
* TableServiceAsyncClientBuilder should be called `TableServiceClientBuilder`. https://azure.github.io/azure-sdk/java_design.html#service-clients